### PR TITLE
feat: add additional config methods to coder sdk

### PR DESCRIPTION
**Pull request by coadler**

### CODE REVIEW

File: coder-sdk/config.go [hunk 1]
```diff
@@ -140,18 +146,81 @@ func (c *DefaultClient) PutSiteConfigExtensionMarketplace(ctx context.Context, r
...
+func (c *DefaultClient) PutSiteConfigDevURLAccess(ctx context.Context, req ConfigDevURLAccess) error {
+	return c.requestBody(ctx, http.MethodPut, "/private/devurls/config", req, nil)
+}
+
+// ConfigSSHSettings is the site configuration for SSH.
+type ConfigSSHSettings struct {
+	KeygenAlgorithm string `json:"keygen_algorithm"`
+}
+
+// SiteConfigSSHSettings fetches the workspace configuration.
```
Comment (deansheather) [inline coder-sdk/config.go:214]: Wrong doc

File: internal/cmd/users_test.go [hunk 2]
```diff
@@ -24,9 +24,9 @@ func Test_users(t *testing.T) {
 
 func assertAdmin(t *testing.T, users []coder.User) {
 	for _, u := range users {
-		if u.Username == "admin" {
+		if u.Username == "kyle" {
 			return
 		}
 	}
-	slogtest.Fatal(t, "did not find admin user", slog.F("users", users))
+	slogtest.Fatal(t, "did not find kyle user", slog.F("users", users))
```
Comment (deansheather) [inline internal/cmd/users_test.go:31]: Did Kyle rename `admin` to `kyle` on master?

File: coder-sdk/config.go [hunk 3]
```diff
@@ -140,18 +146,81 @@ func (c *DefaultClient) PutSiteConfigExtensionMarketplace(ctx context.Context, r
...
-	if err := c.requestBody(ctx, http.MethodGet, "/api/private/config", nil, &conf); err != nil {
+	if err := c.requestBody(ctx, http.MethodGet, "/v0/workspaces/config", nil, &conf); err != nil {
 		return nil, err
 	}
 	return &conf, nil
 }
+
+// PutSiteConfigWorkspaces sets the workspace configuration.
+func (c *DefaultClient) PutSiteConfigWorkspaces(ctx context.Context, req ConfigWorkspaces) error {
+	return c.requestBody(ctx, http.MethodPut, "/v0/workspaces/config", req, nil)
```
Comment (deansheather) [inline coder-sdk/config.go:165]: All of the paths in the existing methods contain the `/api/` prefix.

**Review by deansheather — APPROVED** [review]

File: internal/cmd/users_test.go (hunk 2 above)
Comment (coadler) [reply internal/cmd/users_test.go:31]: The API no longer returns the admin user so I just picked another.

**Pull request merged**